pymeasure/instruments/rigol/rigol_dg800.py

```diff
@@ -118,7 +118,7 @@ class VoltageChannel(Channel):
         ":SOUR{ch}:PULS:DCYC?",
         ":SOUR{ch}:PULS:DCYC %f",
         """
-        Control the duty cycle of pulses out of the waveform generator.
+        Control the duty cycle of pulses out of the waveform generator. (float)
         :param duty_cycle: (float) The pulse's duty cycle
```

Comment:
Now you can remove this line, as the previous line states, that the parameter is a float.
As there is only one parameter which cannot be accessed by name, this line does not give any valuable information to the reader.
```suggestion
```